### ISSUE TITLE
Do you plan to update this R package?

### ISSUE THREAD

**Issue by rsmiller74** [ecohealthalliance/wahis#33]
Now that WAHIS has changed do you plan to update this R package?  It has been very useful in the past.

Thanks!

Ryan

**Comment by Dhihram** [comment]
I agree, I hope this package will be updating, It is very useful for disease surveillance
